content/documentation/getting-started/index.adoc

```diff
@@ -69,36 +69,40 @@ If you prefer to use the latest Kiali version then do not install Kiali during t
 
-==== Default Installation
 
-This option installs the Kiali Operator and the Kiali CR (Custom Resource), with default options.  Kiali will be able to access existing namespaces, but will require a CR update to access future namespaces. See link:#_namespace_management[Namespace Management] for more information. Run the install script via this command:
+==== Default Install
+
+This option installs the Kiali Operator and the Kiali CR, with default options.  This is good for quick installs when the namespaces to be monitored already exist.
+
+When the Kiali CR is created it triggers the operator to then install Kiali. Kiali will be able to access existing namespaces, but it will require a CR update to access future namespaces. See link:#_namespace_management[Namespace Management] for more information. Run the install script via this command:
 
 [source,bash]
 ----
 bash <(curl -L https://git.io/getLatestKialiOperator)
 ----
 
 
-==== Demo/Dev Installation
+==== Demo/Dev Install
+
+This option installs the Kiali Operator and the Kiali CR. It uses a non-default setting for accessible-namespaces making all current and future namespaces accessible to Kiali. This option is good for demo and development installations. This option grants special cluster role permissions and is not recommended for production. See link:#_namespace_management[Namespace Management] for more information.
 
-Like the Default Installation but Kiali will be able to access existing namespaces and future namespaces. Because this grants special cluster role permissions, it is not recommended for production. Run the install script via this command:
+When the Kiali CR is created it triggers the operator to then install Kiali. Run the install script via this command:
 
 [source,bash]
 ----
 bash <(curl -L https://git.io/getLatestKialiOperator) --accessible-namespaces '**'
 ----
 
 
-==== Custom Installation
+==== Operator-Only Install
 
-This option installs the Kiali Operator and a Kiali CR (Custom Resource) with a custom configuration. To generate the CR file it is recommended to copy the fully documented link:https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[example Kiali CR yaml file].  Edit the file, being careful to maintain proper formatting, and save it to a local file (e.g. my-kiali-cr.yaml).
-
-icon:lightbulb[size=1x]{nbsp} It is important to understand the `deployment:accessible_namespaces` setting in the CR.  See link:#_accessible_namespaces[Accessible Namespaces] for more information.
+This option installs only the Kiali Operator.  This option is good when you know there will be a customized Kiali CR.
```

Comment:
This option is good when you plan to customize the Kiali CR?

Reply:
+1